src/components/contact-us/contact-us-controller.ts

```diff
@@ -62,27 +65,58 @@ export function contactUsGet(req: Request, res: Response): void {
   if (req.query.supportType === SUPPORT_TYPE.GOV_SERVICE) {
     return res.render("contact-us/index-gov-service-contact-us.njk");
   }
-  let referer = req.headers.referer;
+
+  let referer = validateReferer(req.headers.referer);
```

Comment:
I know this doesn't make a practical difference here, but should we prefer `req.get('Referrer')` as a slight guard against the referer/referrer spelling issue?: https://www.geeksforgeeks.org/express-js-req-get-function/